### PR TITLE
Sphinx documentation

### PR DESCRIPTION
**Pull request by boyska**
sphinx setup, docstring fix, code reindenting
No real documentation has been added :)

With this you could start publishing documentation on pythonhosted easily, using
`python setup.py upload_sphinx`

### CODE REVIEW

**Comment by boyska** [comment]
@ael-code any news on this?

**Comment by ael-code** [comment]
On Dec 9, 2014 1:18 AM, "BoySka" notifications@github.com wrote:

> @ael-code any news on this?

Not yet, sorry

**Comment by boyska** [comment]
Oh gosh! your new commit conflicts in sooo many places with this pull request :(
please consider undoing it.

**Comment by boyska** [comment]
I removed changes to `fsdb.py`, so this pull request only contains sphinx setup.
There is no indentation fix, no docstring change, etc

File: doc/source/index.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.. fsdb documentation master file, created by
+   sphinx-quickstart on Fri Dec  5 12:43:39 2014.
```
Comment (boyska) [inline doc/source/index.rst:2]: sphinx-quickstart created a first version of this file, but it has been edited manually. So, this is NOT an autogenerated file.

File: doc/source/conf.py [hunk 2]
```diff
@@ -0,0 +1,333 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# fsdb documentation build configuration file, created by
+# sphinx-quickstart on Fri Dec  5 12:43:39 2014.
```
Comment (boyska) [inline doc/source/conf.py:5]: This is the only truly autogenerated file.

**Comment by boyska** [comment]
hey @ael-code , would you like to consider it? :)

**Comment by boyska** [comment]
This (or any other solution to the "have a documentation" problem) would help for
insomnia-lab/libreant#84

**Comment by ael-code** [comment]
thanks for the tip. 
To better understand sphinx setup, I made the implementation on my own and used readthedocs to automatically generate docs-html.

**Comment by boyska** [comment]
great, the important thing is that this library now has html documentation!

**Pull request closed without merging**